app/assets/stylesheets/modules/mfa.css

```diff
@@ -13,3 +13,24 @@
     gap: 1rem;
   }
 }
+
+.mfa__header {
+  display: inline-block;
+  padding-top: 0px !important;
+}
+
+.pill {
+  margin-bottom: 20px;
+  margin-left: 10px;
+  width: 10%;
+  max-width: 320px;
```

Comment:
This seems restrictive. Let's remove the width and let the content set the width of the badge. If the content runs longer, the content of the badge should grow horizontally, not stack vertically.